packages/marketplace/src/components/pages/settings/client-setting.tsx

```diff
@@ -39,12 +39,9 @@ export const ClientSettingsPage: React.FC = () => {
           </div>
           <FormSection>
             <LevelRight>
-              {/* Wrap logout button with a div to fix bug in IE */}
-              <div>
-                <Button dataTest="logout-btn" variant="primary" type="button" onClick={logout}>
-                  Logout
-                </Button>
-              </div>
+              <Button dataTest="logout-btn" variant="primary" type="button" onClick={logout}>
```

Comment:
No longer need to wrap a container outside the button as I've already fixed this alignment issue in `packages/elements/src/styles/vendor/bulma-overrides.scss`